source/d3d8to9.cpp

```diff
@@ -18,7 +18,10 @@ std::ofstream LOG;
 extern "C" Direct3D8 *WINAPI Direct3DCreate8(UINT SDKVersion)
 {
 #ifndef D3D8TO9NOLOG
-	LOG.open("d3d8.log", std::ios::trunc);
+	if (!LOG.is_open())
+	{
+		LOG.open("d3d8.log", std::ios::trunc);
+	}
 
 	if (!LOG.is_open())
 	{
```

Comment:
You're right, sorry. i thought that this function was related to those GTA issues reported earlier. Anyway if in some cases this function is called multiple times on startup, I think it's good to limit displaying of the message box to one time.